packages/sdk/contractkit/src/contract-cache.ts

```diff
@@ -193,6 +193,11 @@ export class WrapperCache implements ContractCacheType {
   getMultiSig(address: string) {
     return this.getContract(CeloContract.MultiSig, address)
   }
+
+  getPaymentDelegation() {
```

Comment:
This file is for contract wrapper getters, right? And we already have an `Accounts` getter, so this is probably unnecessary.